src/components/clusterWizard/ClusterWizardStep.css

```diff
@@ -0,0 +1,7 @@
+.cluster-wizard-step .pf-c-wizard__nav-item {
+  white-space: nowrap;
+}
+
+.wizard-nav-item-warning-icon {
+  margin-left: var(--pf-global--spacer--sm);
```

Comment:
`&nbsp;` between the text and icon has the same effect AFAIK

Reply:
right, will do. But this is more PF-firendly way I think.